### PR TITLE
Transactions

### PR DESCRIPTION
**Pull request by pakokrew**

### CODE REVIEW

File: app/src/App.js [hunk 1]
```diff
@@ -43,20 +36,9 @@ function App() {
...
-        return <Transfer />
-      default:
-        return null
-    }
-  }
-
   return (
     <Main theme={appearance}>
-      <Header primary="Transactions" />
+      <Header primary="Transactfdsfons" />
```
Comment (sembrestels) [inline app/src/App.js]: Lol!

File: app/src/lib/token-utils.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ export async function createTokenEVMScript(accounts, tokenManagerAddress) {
  * @returns {Array<string>} array with the [<whole>, <decimal>] parts of the number
  */
 function splitDecimalNumber(num) {
-  const [whole = '', dec = ''] = num.split('.')
+  const [whole = '', dec = ''] = String(num).split('.')
```
Comment (sembrestels) [inline app/src/lib/token-utils.js]: We should receive a string. We should trace why we receive a number here.

Comment (pakokrew) [reply app/src/lib/token-utils.js]: values comes directly from input[number] here, maybe we can cast earlier ? or use decimals.js 😇

Comment (sembrestels) [reply app/src/lib/token-utils.js]: Ok, if there is no conversion to number in the meanwhile we are probably fine. I would cast the number to string just as you are doing it.

**Comment by pakokrew** [comment]
Closing to use the branch in this repo directly

**Pull request closed without merging**